pilot/pkg/kube/admit/admit.go

```diff
@@ -269,8 +273,8 @@ func (ac *AdmissionController) register(client clientadmissionregistrationv1beta
 						admissionregistrationv1beta1.Update,
 					},
 					Rule: admissionregistrationv1beta1.Rule{
-						APIGroups:   []string{model.IstioAPIGroup},
-						APIVersions: []string{model.IstioAPIVersion},
+						APIGroups:   groups,
```

Comment:
Each rule in the array of rules must refer to a valid group/version/resource tuple, e.g.

```golang
var rules []admissionregistrationv1beta1.Rule
for _, schema := range ac.options.Descriptor {
    rules = append(rules, admissionregistrationv1beta.Rule{
        APIGroups: []string{crd.ResourceGroup(&schema)},
        APIVersion: []string{schema.Version},
        Resources: []string{crd.ResourceName(schema.Plural)}, 
    }
}

// assign `rule` in webhook.Webhooks[0].Rules
```

Reply:
But the field is a single Rule: instead of an array of Rules.

Reply:
Woops, I used the wrong type. See line 270 and `admissionregistrationv1beta1.RuleWithOperations`